wgsl/index.bs

```diff
@@ -2747,6 +2747,34 @@ TODO(dsinclair): Add Level-of-Detail via explicit gradient. "Grad" image operand
 TODO(dsinclair): Need gather operations
 </pre>
 
+## Subgroup built-in functions ## {#subgroup-builtin-functions}
+
+<table class='data'>
+  <thead>
+    <tr><td>Subgroup built-in functions<td>SPIR-V
+  </thead>
+  <tr><td>subgroup_size() -&gt; u32<td>SubgroupSize
+  <tr><td>subgroup_local_index() -&gt; u32<td>SubgroupLocalInvocationId
```

Comment:
These two appear to be builtin's in SPIR-V, not function calls.

Reply:
Moved, but I am not sure if things could have better tidy